src/js/addDelete.test.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import handleDelete from './handleDelete.js';
+import addTask from './add.js';
+
+describe('addTask', () => {
+  test('should add a new task to the task and update the local storage', () => {
```

Comment:
- [ ] The command to run the test is not running I wonder why @henask12, check the record below
![5](https://github.com/henask12/To-do-list/assets/77843050/9e42c803-b63e-4bab-92ae-e72af376cab2)

Reply:
HI @joskalenda 

Please Use npm test or npx jest for testing